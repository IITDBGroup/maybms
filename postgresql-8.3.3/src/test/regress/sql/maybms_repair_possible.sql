--test for the combination of possible and repair-key

create table r(a int, b int);

insert into r values (1,2), (1,3);

create table t as repair key(a) in r;

select * from t;

repair key (a) in (select possible a,b from t);

drop table r;
drop table t;
