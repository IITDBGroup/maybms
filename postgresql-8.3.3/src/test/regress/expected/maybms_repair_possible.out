--test for the combination of possible and repair-key
create table r(a int, b int);
insert into r values (1,2), (1,3);
create table t as repair key(a) in r;
select * from t;
 a | b | _v0 | _d0 | _p0 
---+---+-----+-----+-----
 1 | 2 |   1 |   2 | 0.5
 1 | 3 |   1 |   1 | 0.5
(2 rows)

repair key (a) in (select possible a,b from t);
 a | b | _v0 | _d0 | _p0 
---+---+-----+-----+-----
 1 | 2 |   2 |   3 | 0.5
 1 | 3 |   2 |   4 | 0.5
(2 rows)

drop table r;
drop table t;
